docs/_pages/faq.md

```diff
@@ -32,7 +32,7 @@ include_prefix: ../
 
 <h2 id="where-should-i-use">Where should I use Material Design Lite (MDL)?</h2>
 
-If you’re interested in a [Material Design](www.google.com/design/spec/material-design/introduction.html) experience using vanilla Web technologies like CSS, JavaScript and HTML, MDL might be a useful option to consider. We optimise for websites heavy on content, such as marketing pages, articles, blogs and general web content that isn’t particularly app-y.  If you just want to pick some colors, customise a template and ship a Material experience, we try to help make that process simpler.
+If you’re interested in a [Material Design](http://www.google.com/design/spec/material-design/introduction.html) experience using vanilla Web technologies like CSS, JavaScript and HTML, MDL might be a useful option to consider. We optimise for websites heavy on content, such as marketing pages, articles, blogs and general web content that isn’t particularly app-y.  If you just want to pick some colors, customise a template and ship a Material experience, we try to help make that process simpler.
```

Comment:
Oh right.

#### Start troll

Why isn't Google redirecting that on its own again?

#### end troll